content/docs/start/experiment-management/experiments.md

```diff
@@ -90,6 +90,13 @@ models, plots, and metrics in the respective directories. The experiment is then
 associated with the values found in the parameters file (`params.yaml`) and
 other dependencies, as well as the metrics produced.
 
+<admon type="tip">
+
+Current workspace status can also be saved as an experiment using
+[`dvc exp save`](/doc/command-reference/exp/save).
```

Comment:
I'd phrase this "If you already run your code manually you can capture..."

Also, no need to manually link commands (they're auto-linked when in code blocks) 🙂